test/pylib/rest_client.py

```diff
@@ -233,6 +233,13 @@ async def enable_injection(self, node_ip: str, injection: str, one_shot: bool, p
         await self.client.post(f"/v2/error_injection/injection/{injection}",
                                host=node_ip, params={"one_shot": str(one_shot)}, json={ key: str(value) for key, value in parameters.items() })
 
+    async def disable_injection(self, node_ip: str, injection: str) -> None:
+        """Enable error injection named `injection` on `node_ip`. Depending on `one_shot`,
```

Comment:
This probably-copy-pasted comment is wrong :-(

Reply:
thanks.  will fix.

Reply:
dropped in v2.
it turns out we already had such a function